src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformEditView.tsx

```diff
@@ -462,7 +462,7 @@ export default function TerraformEditView(
                           )}
                           <div className={css.divider} />
                           <TfVarFileList
-                            formik={formik}
+                            formik={formik as any}
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
                                    formik={formik as unknown}
```